sormas-api/src/main/java/de/symeda/sormas/api/person/PersonDto.java

```diff
@@ -466,6 +466,9 @@ public void setDeathDate(Date deathDate) {
 	private void setPersonContactInformation(String contactInfo, PersonContactDetailType personContactDetailType) {
 		for (PersonContactDetailDto contactDetailDto : getPersonContactDetails()) {
 			if (contactDetailDto.getPersonContactDetailType() == personContactDetailType && contactDetailDto.isPrimaryContact()) {
+				if (contactInfo.equals(contactDetailDto.getContactInformation())){
```

Comment:
Code formatting; apart from that, everything's fine, you can merge the PR after applying code formatting